tests/installation/install_and_reboot.pm

```diff
@@ -99,6 +99,10 @@ sub run {
             assert_screen \@tags, $timeout;
         }
 
+        if (match_has_tag("yast2_error")) {
+            die "An YaST error occurred. Check the logs.";
```

Comment:
Please, fix typo "An YaST..." to "A YaST..."

Reply:
sorry but that still applies

Reply:
True. I guess I missed that during the rebase.

Reply:
@cwh42 please fix that typo, then LGTM